auth-api/src/auth_api/services/org.py

```diff
@@ -397,6 +334,7 @@ def update_org(self, org_info, origin_url: str = None):  # pylint: disable=too-m
 
         # Update mailing address Or create new one
         if mailing_address:
+            has_org_updates = True
```

Comment:
if mailing address is getting updated , why set has_org_updates=True? it goes to different table rite?

Reply:
There are two methods where we call this add_contact_to_org, create_org and update_org 
In the existing system, we call save at the end of create_org and update_org  that also saves contact_links. 
Is there other alternatives for this?